setup.py

```diff
@@ -37,10 +37,13 @@
     packages=['wfsim',
               'wfsim.pax_datastructure',],
     classifiers=[
-        'Development Status :: 2 - Pre-Alpha',
+        'Development Status :: 4 - Beta',
```

Comment:
another status may also be applicable so please correct me if I am wrong:
https://pypi.org/classifiers/